### ISSUE TITLE
Support AWS EC2: Initial Updates & Onboarding

### ISSUE THREAD

**Issue by monicapatel21** [wri/wri_sites#125]
See if there's an update? Bob was supposed to talk to ben about best ways to update these ([Rows 11 & 12](https://docs.google.com/spreadsheets/d/1bp2jAzgozwQTECs63LzSvVJqTU4EOC4qnECNC1TCLAI/edit#gid=1617391067))

- [ ] transitbuses.wri.org (transit-buses on EC2) | (more server maintenance, than application maintenance) SSL issues. which is an immediate functionality problem; no source code / if lucky: readme with info about lost repos // Simple apps that generate output consumed elsewhere / an OS on EC2 needs to be patched for security (not actively receiving data updates -- if they're pulling new data its because they're pulling from a Google Sheet a researcher is updating) |   | AWS EC2

- [ ] ev-simulator.wri.org (sim-prod2 on EC2) | (more server maintenance, than application maintenance) SSL issues. which is an immediate functionality problem; no source code / if lucky: readme with info about lost repos // Simple apps that generate output consumed elsewhere / an OS on EC2 needs to be patched for security (not actively receiving data updates -- if they're pulling new data its because they're pulling from a Google Sheet a researcher is updating) |   | AWS EC2

**Comment by monicapatel21** [comment]
related to #124

**Comment by monicapatel21** [comment]
Add us as an organization so we can get the access to review these sites (to AWS). We need to coordinate with Lev on MFA.

**Comment by monicapatel21** [comment]
Bob and Ben discussing best options for providing support for these. I'll check in with the client about adding us as an organization to review the sites.

**Comment by StudioZut** [comment]
@katieescoto can you get the SSL certs updated? If it's not straight-forward we could consider setting up via cloudfront with AWS ACM for the cert.

**Comment by katieescoto** [comment]
@StudioZut Here is an overall summary of where we are for each app:

- EV Simulator - works now. I don’t think it was working last time I dug into this, so if it was anything I did it was probably just making sure the DNS IP matched the one in AWS. The SSL certificate expired January 2022, so I requested a new one a couple days ago. In the ACM dashboard, its status is “pending validation”. But hopefully once it’s validated, navigating there won’t result in the “this site is unsafe” error.
- Transit Buses - I can get it to work by executing the java application in the server. This isn’t a permanent solution because it terminates once I log out of the instance, so I need to dig into the files that were created in systemd to run it as a service. My understanding based on a lot of googling is that this is the best way to ensure it’s running continuously. This site already has a valid SSL, which I imported to AWS and seems to be working. There is also some work to be done on the DNS records for this app, because right now, the application runs on port 8000 and only in HTTP mode.

For the SSLs specifically, I've requested a new certificate for EV Simulator on Tuesday and it's currently pending validation. I'll keep monitoring and make sure that once it's validated, the DNS records are accurate and the app loads securely. The SSL for Transit Buses is valid until August and has been imported to AWS but I think there's some work to be done in Cloudflare to make sure it's loading properly.
